packages/rum-slim/src/domain/getSessionReplayLink.spec.ts

```diff
@@ -0,0 +1,29 @@
+import type { RumConfiguration } from '@datadog/browser-rum-core'
+import { getSessionReplayLink } from './getSessionReplayLink'
+
+const DEFAULT_CONFIGURATION = {
+  site: 'datad0g.com',
+} as RumConfiguration
+
+describe('getReplayLink (slim package)', () => {
+  const parameters: Array<[string, string | undefined, string]> = [
+    ['datadoghq.com', undefined, 'app.datadoghq.com'],
+    ['datadoghq.com', 'toto', 'toto.datadoghq.com'],
+    ['datad0g.com', undefined, 'dd.datad0g.com'],
+    ['datad0g.com', 'toto', 'toto.datad0g.com'],
+    ['us3.datadoghq.com', undefined, 'us3.datadoghq.com'],
+    ['us3.datadoghq.com', 'toto', 'toto.us3.datadoghq.com'],
+    ['us5.datadoghq.com', undefined, 'us5.datadoghq.com'],
+    ['us5.datadoghq.com', 'toto', 'toto.us5.datadoghq.com'],
+  ]
+
+  parameters.forEach(([site, subdomain, host]) => {
+    it(`should return ${host} for subdomain "${
+      subdomain ?? 'undefined'
+    }" on "${site}" with query params if view is found`, () => {
+      const link = getSessionReplayLink({ ...DEFAULT_CONFIGURATION, site, subdomain })
+
+      expect(link).toBe(`https://${host}/rum/replay/sessions/session-id?error-type=slim-package`)
+    })
+  })
+})
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏You have tests for `getDatadogOrigin`. You might not need to test for every case in other spec files. Maybe just keep a single test like:

```suggestion
describe('getReplayLink (slim package)', () => {
  it('should return the replay link with a "slim-package" error type', () => {
      const link = getSessionReplayLink(DEFAULT_CONFIGURATION)
      expect(link).toBe(`https://dd.datad0g.com/rum/replay/sessions/session-id?error-type=slim-package`)
  })
})
```